frontend/rolecall/src/app/services/super-ballet-display.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+
+@Injectable({providedIn: 'root'})
+export class SuperBalletDisplayService {
+  superBalletIsOpenMap = new Map<string, boolean>();
+
+  changeOpenState(uuid: string, isOpen: boolean) {
+    this.superBalletIsOpenMap.set(uuid, isOpen);
+  }
+
+  isOpen(uuid: string): boolean {
+    return Boolean(this.superBalletIsOpenMap.get(uuid));
+  }
+
+  removeFromDisplayList(uuid: string) {
+    if (this.superBalletIsOpenMap.has(uuid)) {    
+      this.superBalletIsOpenMap.delete(uuid);
+    }    
+  }
+
+  verifyIsInDisplayList(uuid: string, isOpen: boolean) {
```

Comment:
Need to discuss this one, naming isn't right and I'm not clear about purpose. Something with a prefix of "verify" should not have any state side-effects, I would expect it to either throw an exception if verification fails, or possibly return a boolean.

Reply:
Done.